tests/integration/goal_oriented/test_scrape.py

```diff
@@ -0,0 +1,99 @@
+import concurrent
+import os
+import unittest
+
+import vcr
+import requests
+
+from autogpt.agent import Agent
+from autogpt.commands.command import CommandRegistry
+from autogpt.commands.file_operations import delete_file, read_file
+from autogpt.commands.web_requests   import scrape_text, scrape_links
+from autogpt.config import AIConfig, Config
+from autogpt.memory import get_memory
+from tests.integration.goal_oriented.vcr_helper import before_record_request
+from tests.utils import requires_api_key
+
+current_file_dir = os.path.dirname(os.path.abspath(__file__))
+
+my_vcr = vcr.VCR(
+    cassette_library_dir=os.path.join(current_file_dir, "cassettes"),
+    record_mode="new_episodes",
+    before_record_request=before_record_request,
+)
+
+CFG = Config()
+
+@requires_api_key("OPENAI_API_KEY")
+def test_browse_website(workspace) -> None:
+    CFG.workspace_path = workspace.root
+    CFG.file_logger_path = os.path.join(workspace.root, "file_logger.txt")
+
+    file_name = str(workspace.get_path("test_scrape.txt"))
+    agent = create_writer_agent(workspace)
+    try:
+        with my_vcr.use_cassette(
+            "scrape_text.vcr.yml",
+            filter_headers=[
+                "authorization",
+                "X-OpenAI-Client-User-Agent",
+                "User-Agent",
+            ],
+        ):
+            with concurrent.futures.ThreadPoolExecutor() as executor:
+                future = executor.submit(agent.start_interaction_loop)
+                try:
+                    result = future.result(timeout=40)
+                except concurrent.futures.TimeoutError:
+                    assert False, "The process took longer than 40 seconds to complete."
+    # catch system exit exceptions
+    except SystemExit:  # the agent returns an exception when it shuts down
+        content = read_file(file_name)
+        assert "£25.89" in content, f"Expected £25.89, got {content}"
+
+
+def create_writer_agent(workspace):

```

Comment:
rename it create_browse_website_agent

Reply:
[38cbc8c](https://github.com/Significant-Gravitas/Auto-GPT/pull/3261/commits/38cbc8c330d6d52b00aaa132568d1dfc6a76c3d2)

Changed create_writer_agent  to create_browse_agent (line 33 and 55 in the file) as requested.